lib/tracker/mode_s/decode_test.go

```diff
@@ -480,3 +480,18 @@ func TestFrame_decode13bitAltitudeCode(t *testing.T) {
 		})
 	}
 }
+
+func BenchmarkFrame_DecodeDF0(b *testing.B) {
+	t := time.Now()
+	var ok bool
+	for i := 0; i < b.N; i++ {
```

Comment:
Recent Go versions suggest using `for b.Loop() {`

Reply:
I did go looking for this, but when I realised I did not yet have the latest golang...

Reply:
also, we have a LOT of these sorts of benchmarking loops